readme.txt

```diff
@@ -36,6 +36,9 @@ Crouton was forked from BMLT Tabbed UI plugin in 2018.  This plugin provides a T
 
 == Changelog ==
 
+= 3.5.1 =
+* Added checkbox for using tomato as root server. [#181]
+
```

Comment:
3.5.0 isn't released yet.  Can you adjust?

Reply:
oops sure thing